CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ set(USE_EXTERNAL_LLVM ON)
 include("GNUInstallDirs")
 set(LLVM_DIR_ORIG ${LLVM_DIR})
 set(Clang_DIR_ORIG ${Clang_DIR})
+add_definitions(-DBINDER_VERSION_STRING=\"${VERSION}\")
```

Comment:
@andriish please note that corresponding changes will need to be added to source/CMakeLists.txt

Reply:
Yes.